plugin/discovery/get.go

```diff
@@ -28,6 +27,12 @@ import (
 
 const protocolVersionHeader = "x-terraform-protocol-version"
 
+const gpgVerificationError = `GPG signature verification error:
+Terraform was unable to verify the GPG signature of the downloaded provider
+files using the keys downloaded from the Terraform Registry. This may mean that
+the publisher of the provider removed the key it was signed with, or that the
+distributed files were changed after this version was released`
```

Comment:
nitpick: Can we please add the missing period to the end of this sentence?

Reply:
Sure! I omitted it because originally the string was inlined in `fmt.Errorf` and golint complained that it ended in a period. Also in the output shown to the user in the screenshot above, the caller is adding the period onto the end of the error. BUT I now see that the other place it's printed later on does not have it 😄.

Either way, will get it fixed in both places, thanks!

Reply:
This particular codepath is currently unfortunately in a kinda "in-between" state between the way we formerly dealt with errors to the user (just printing out huge blocks of red text) and the new way (using diagnostics objects), so in situations like this things get a little weird because our bridge code is converting this into a diagnostic just in time to print it and we have some different notational conventions for diagnostics than Go does for errors.

With that said, I think it's okay for things to be a little "off" here for the moment and we'll continue to rework the error handling for `terraform init` to do it all via the newer mechanism. In the long run this probably won't be a Go `error` at all but rather a `tfdiags.Diagnostic`, and so I think we should ignore golint's advice for now under the assumption that we'll be using this string in a different way before too long.

Reply:
Added the trailing period!